addons/website/static/src/js/content/menu.js

```diff
@@ -814,6 +815,39 @@ publicWidget.registry.MegaMenuDropdown = publicWidget.Widget.extend({
         this.options.wysiwyg?.odooEditor.observerActive("moveMegaMenu");
     },
 
+    /**
+     * @private
+     */
+    _updateActiveMenuLinks() {
+        // Prevent having several active links in the menu.
+        if (this.el.querySelector(".navbar:nth-child(1) a.nav-link.active")) {
```

Comment:
```suggestion
        if (this.el.querySelector(".navbar #top_menu a.nav-link.active")) {
```
- no need of `:nth-child(1)` because you use `querySelector` and not `querySelectorAll`.
- I would add `#top_menu` because `nav-link` could be used at other locations in the header (e.g. snippet dropped inside the header, custo, etc.)